eox_tenant/models.py

```diff
@@ -91,16 +91,16 @@ def get_microsite_for_domain(cls, domain):
         return microsites[0] if microsites else None
 
     @classmethod
-    def get_value_for_org(cls, org, val_name, current_value):
+    def get_value_for_org(cls, org, val_name, current_tenant):
         """
         Filter the value by the org in the values field.
-        If any value from an org matches the current_value, that value is returned.
+        Value from current tenant is prioritized if org is present.
```

Comment:
Nit: It might be useful to mention the scenario where this is required, i.e. when same org is configured for multiple tenants/microsites